src/widget/loginscreen.cpp

```diff
@@ -31,6 +31,16 @@
 #include <QMessageBox>
 #include <QDebug>
 
+#ifdef Q_OS_WIN32
+    #include <windows.h>
+#else
+    #include <X11/XKBlib.h>
```

Comment:
Err, this needs to be fixed:

* it's broken on osx
* introduction of a hard dep on xorg is a no-go – there's a define for ~that: https://github.com/tux3/qTox/blob/4432ab6398aa4c9cdc222cf46170b820b9e33fa0/qtox.pro#L127